api/src/aggregator/aggregator.controller.ts

```diff
@@ -71,6 +71,15 @@ export class AggregatorController {
     })
   }
 
+  @Get(':address')
+  async findByAddress(@Param('address') address: string) {
+    return await this.aggregatorService.findAll({
```

Comment:
I think this should use `findOne` rather than `findAll`. Do we expect that there are more aggregators with the same address? I think this is the reason why it returned an array in another snippet of this PR.